### PR TITLE
[4.0] Fix icon width in dropdowns etc

### PR DESCRIPTION
**Pull request by brianteeman**
This simple PR updates the scss so that the styling for icomoon and fontawesome (icon- and fa-) are the same

Can be tested on the actions dropdown with the batch button

### before
image

### after
image

### CODE REVIEW

File: build/media_source/system/scss/_icomoon.scss [hunk 1]
```diff
@@ -1,19 +1,24 @@
 /* IcoMoon Conversion */
 
 [class^="icon-"]:not(.input-group-text),
-[class*=" icon-"]:not(.input-group-text) {
+[class*=" icon-"]:not(.input-group-text),
+[class^="fa-"]:not(.input-group-text),
+[class*=" fa-"]:not(.input-group-text)
+ {
```
Comment (Quy) [inline build/media_source/system/scss/_icomoon.scss]: Move to the previous line.

File: administrator/templates/atum/scss/blocks/_form.scss [hunk 2]
```diff
@@ -148,7 +148,10 @@ legend {
 }
 
 [class^="icon-"]:not(.input-group-text),
-[class*=" icon-"]:not(.input-group-text) {
+[class*=" icon-"]:not(.input-group-text),
+[class^="fa-"]:not(.input-group-text),
+[class*=" fa-"]:not(.input-group-text)
+ {
```
Comment (Quy) [inline administrator/templates/atum/scss/blocks/_form.scss]: Move to previous line.

**Comment by C-Lodder** [comment]
The issue derives from: https://github.com/joomla/joomla-cms/blob/4.0-dev/administrator/templates/atum/scss/vendor/bootstrap/_dropdown.scss#L8-L13

It should probably be removed instead

**Comment by brianteeman** [comment]
@C-Lodder you're the expert but are you sure its not because of the    
` *margin-right: .3em;
`

**Comment by C-Lodder** [comment]
@brianteeman The `*` before a property is a IE hack...specifically for IE7 if I rightly remember.

The should be removed too but out of scope for the PR

**Comment by brianteeman** [comment]
sorry I meant the margin-right 0.5

**Comment by brianteeman** [comment]
@C-Lodder I have updated it - can you confirm its ok now

**Comment by C-Lodder** [comment]
@brianteeman you should just be able to remove the width and revert your initial changes. It will no doubt effect something else in the CMS but that can be addressed later with a proper fix once found

**Comment by brianteeman** [comment]
by doing both it avoids the chance of effects elsewhere doesnt it?

**Comment by C-Lodder** [comment]
Meh I suppose so

**Comment by ghost** [comment]
Please resolve conflicting files so this PR can get tested at Worldwide Pizza, Bugs & Fun, October 19th

**Comment by brianteeman** [comment]
@franz-wohlkoenig I really hope it will be tested and merged before then

**Comment by Nilesh5995** [comment]
I have tested this item :red_circle: unsuccessfully on 2f8e5be16db3ebb6f351eacbc0df6f1d5c093974<br /><br />Testing in the chrome browser.
It looks like shown in before image. Link - https://user-images.githubusercontent.com/1296369/63854445-17e98200-c995-11e9-843a-9f9b68b7f7d4.png<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26056">issues.joomla.org/tracker/joomla-cms/26056</a>.</sub>

**Comment by ahghatol** [comment]
I have tested this item :white_check_mark: successfully on 2f8e5be16db3ebb6f351eacbc0df6f1d5c093974<br /><br />Worked<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26056">issues.joomla.org/tracker/joomla-cms/26056</a>.</sub>

**Comment by riishede** [comment]
I have tested this item 🔴 unsuccessfully on 2f8e5be16d

php: Linux web10 4.1.42-vs2.3.8.6-beng #1 SMP Fri Jul 28 13:09:21 BST 2017 x86_64
dbserver: mysql
dbversion: 5.6.32-1~dotdeb+7.1
dbcollation: latin1_swedish_ci
dbconnectioncollation: utf8mb4_general_ci
phpversion: 7.2.16
server: Apache
sapi_name: cgi-fcgi
version: Joomla! 4.0.0-beta1-dev Development [ Amani ] 17-October-2019 20:21 GMT
useragent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:69.0) Gecko/20100101 Firefox/69.0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26056">issues.joomla.org/tracker/joomla-cms/26056</a>.</sub>

**Comment by Schmidie64** [comment]
I have tested this item :red_circle: unsuccessfully on 2f8e5be16db3ebb6f351eacbc0df6f1d5c093974<br /><br />After activating the patch, no change can be detected.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26056">issues.joomla.org/tracker/joomla-cms/26056</a>.</sub>

**Comment by brianteeman** [comment]
To everyone saying this did not work did you run "npm i" after applying the patch to rebuild the css

**Comment by Schmidie64** [comment]
strangely, actually i had npm i run. Now it's working.

**Comment by Schmidie64** [comment]
I have tested this item :white_check_mark: successfully on 2f8e5be16db3ebb6f351eacbc0df6f1d5c093974<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26056">issues.joomla.org/tracker/joomla-cms/26056</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26056">issues.joomla.org/tracker/joomla-cms/26056</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Comment by infograf768** [comment]
IMHO, this should be reverted as it breaks a lot of stuff for quickicons and dashboards.
See https://github.com/joomla/joomla-cms/issues/26741

**Comment by infograf768** [comment]
Please test https://github.com/joomla/joomla-cms/pull/26757

**Pull request merged**